load_confounds/tests/test_strategies.py

```diff
@@ -77,26 +77,27 @@ def test_Scrubbing():
         "csf_power2",
         "csf_derivative1_power2",
         "white_matter_derivative1",
-        "motion_outlier_0",
-        "motion_outlier_1",
     ]
 
     for check in list_check:
         assert check in conf.columns_
+    # should have six motion outliers from scrubbing
+    assert sum("motion_outlier_" in col for col in conf.columns_) == 8
+    # assert len(conf.sample_mask_) == 24
```

Comment:
not sure why this is commented out.

Reply:
In the test, I tested method `load` rather than `load_mask`, so `sample_mask_` was not returned. As we are going to return `sample_mask` as the default for the scrubbing strategy, this test will be rewritten.